recruiter/cli.py

```diff
@@ -1,11 +1,11 @@
 import json
 
 from argparse import ArgumentParser
-from candidate import Candidate
+from recruiter.candidate import Candidate
```

Comment:
absolute paths are the recommended standard per PEP-8

https://www.python.org/dev/peps/pep-0008/

Ctrl+F for `Absolute imports are recommended`